instrumentation/graphql/lib/opentelemetry/instrumentation/graphql/tracers/graphql_trace.rb

```diff
@@ -87,10 +88,11 @@ def execute_field_lazy(field:, query:, ast_node:, arguments:, object:, &block)
             platform_key = _otel_execute_field_key(field: field)
             return super unless platform_key
 
-            attributes = {}
-            attributes['graphql.field.parent'] = field.owner&.graphql_name
-            attributes['graphql.field.name'] = field.graphql_name
-            attributes['graphql.lazy'] = true
+            attributes = {
+              'graphql.field.parent' => field.owner&.graphql_name,
```

Comment:
Something I did not notice before is that this line and `owner` may be `nil`.

That will result in a noisy error handling when trying to add the attributes to the span: 

https://github.com/open-telemetry/opentelemetry-ruby/blob/37e225ac63fef4617fc78bf1f638417eeb6a6e91/sdk/lib/opentelemetry/sdk/internal.rb#L57

A `compact!`, or equivalent, would be appreciated before sending this along to `in_span`